### PR TITLE
dts: sdm450: Add HTC Desire 12+ (brepdugl) Support

### PR DESCRIPTION
**Pull request by Hinokami-Kagura**
Checked that the phone is able to boot into lk2nd, however that's phone can't found any kernel source...
image

### CODE REVIEW

File: dts/sdm450-htc-brepdugl.dts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ 
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8952.dtsi"
```
Comment (alikates) [inline dts/sdm450-htc-brepdugl.dts:6]: Is this correct?

Comment (Hinokami-Kagura) [reply dts/sdm450-htc-brepdugl.dts:6]: use msm8952.dtsi, but does it have different with msm8953.dtsi? thanks!

Comment (ultra-azu) [reply dts/sdm450-htc-brepdugl.dts:6]: They are the same, but you should use msm8953.dtsi anyways, 8952 is another SOC.

File: dts/rules.mk [hunk 2]
```diff
@@ -18,6 +18,7 @@ DTBS += \
 	$(LOCAL_DIR)/msm8953-xiaomi-ysl.dtb \
 	$(LOCAL_DIR)/sdm450-samsung-r04.dtb \
 	$(LOCAL_DIR)/sdm450-samsung-r05.dtb \
+  $(LOCAL_DIR)/sdm450-htc-brepdugl.dtb \
```
Comment (barni2000) [inline dts/rules.mk:21]: Please fix the indentation

**Review by barni2000 — CHANGES_REQUESTED** [review]
Please squash your commits together

**Comment by M0Rf30** [comment]
committed on main

**Pull request closed without merging**